test/plans/upstream.fmf

```diff
@@ -0,0 +1,25 @@
+summary: Run browser integration tests on the host
+prepare:
+  - name: Install cockpit and test dependencies
+    how: install
+    package:
+    - cockpit
+    - cockpit-ws
+    - cockpit-system
+    - cockpit-bridge
+    - git
+    - libvirt-python3
+    - make
+    - npm
+    - python3
+  - name: Install Chromium
+    script: ./prepare-chromium.sh
```

Comment:
@jscotka: Does that work for you? I tried this locally with `tmt run`, and it is *entirely* ignored.

Reply:
See https://github.com/martinpitt/cockpit/tree/tmt-split-setup

Reply:
Hi, no, there were issue, that it was not able to find this local path and had to be workarounded by some `../../` magic
I remember, we've done some issue and PR's exacly to this issue commected with this cockpit PR, because I've met it and reported.
But it should be fixed  in upstream, because there is the test what I remember: https://github.com/psss/tmt/tree/master/tests/prepare/shell/data
so maybe try to install/upgrade to new TMT, not sure if the release containing this feature is already shipped to channel what you use for your installation

Reply:
yep, your changes in https://github.com/martinpitt/cockpit/tree/tmt-split-setup seems very well. it is exactly What I've expected.